assembl/static2/js/app/utils/translations.js

```diff
@@ -462,6 +463,7 @@ const Translations = {
       confirmDeletionBody: 'Are you sure you want to delete this message?',
       confirmDeletionButtonDelete: 'Delete',
       confirmDeletionButtonCancel: 'Cancel',
+      whatWeNeedToKnow: 'What we need to know',
```

Comment:
In English, often we wouldn't say "What we need to know", it would likely say, "What you need to know". This is a design choice though. @FrankEscoubes @paolina92, thoughts?